src/Client.php

```diff
@@ -333,6 +334,10 @@ private function buildPayload( $type, $to, $templateId, array $personalisation,
             $payload['reference'] = $reference;
         }
 
+        if ( isset($emailReplyToId) && $emailReplyToId != '' ) {
```

Comment:
!empty() instead?

Reply:
Its consistent with the other checks in that method so I'll leave it.